container/container.go

```diff
@@ -8,14 +8,23 @@ import (
 	"errors"
 	"net"
 	"net/http"
+	"regexp"
 	"strings"
 	"time"
 
 	"github.com/fsouza/go-dockerclient"
 	"github.com/hashicorp/golang-lru"
 )
 
-var ErrTsuruVariablesNotFound = errors.New("could not find wanted envs")
+var (
+	ErrTsuruVariablesNotFound = errors.New("could not find wanted envs")
+
+	hexRegex          = regexp.MustCompile(`(?i)^[a-f0-9]+$`)
+	processNameLabels = []string{"bs.tsuru.io/log-app-name", "log-name", "io.kubernetes.pod.name"}
```

Comment:
Shouldn't the second item be `log-app-name`?

Reply:
fixed!